### PR TITLE
Allow webpack@3

### PR DESCRIPTION
**Pull request by vramana**

### CODE REVIEW

**Comment by mastilver** [comment]
@vramana Thank you for your PR, please see my comment there: https://github.com/jantimon/html-webpack-plugin/issues/699#issuecomment-309430315

File: package.json [hunk 1]
```diff
@@ -63,6 +63,6 @@
     "toposort": "^1.0.0"
   },
   "peerDependencies": {
-    "webpack": "1 || ^2 || ^2.1.0-beta || ^2.2.0-rc"
+    "webpack": "1 || ^2 || ^2.1.0-beta || ^2.2.0-rc" || ^3
```
Comment (mastilver) [inline package.json]: close the string

**Comment by vramana** [comment]
@mastilver Fixed it.

**Comment by vramana** [comment]
It seems #706 and #707 does a better job than my PR. Hence I am closing my PR

**Pull request closed without merging**